src/code/z_sub_s.c

```diff
@@ -525,9 +565,92 @@ void SubS_DrawShadowTex(Actor* actor, GameState* gameState, u8* tex) {
     CLOSE_DISPS(gfxCtx);
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013D0E0.s")
+/**
+ * Computes the rotation based on the options and target rotation value
+ *
+ * @param[in,out] rot the computed rotation
+ * @param[in] rotMax the max rotation in binary angles
+ * @param[in] target the target rotation value
+ * @param[in] slowness how slow to rotate, the larger the number the slower the rotation
+ * @param[in] rotAdjMin the minimun rotation adjustment in degrees
+ * @param[in] rotAdjMax the maximum rotation adjustment in degrees
+ */
+s16 SubS_ComputeTurnToPointRot(s16* rot, s16 rotMax, s16 target, f32 slowness, f32 rotAdjMin, f32 rotAdjMax) {
+    s16 prevRot = *rot;
+    f32 rotAdj;
+    f32 prevRotAdj;
+
+    rotAdj = (f32)(target - *rot) * (360.0f / 65536.0f);
```

Comment:
In general I prefer these written as `(f32)0x10000` (do we not have a macro for these?)

Reply:
Not yet, just the camera specific ones. There was a little discussion in one of the math threads, and I think we were waiting for sys_matrix/skin_matrix to be updated / waiting for OOT to figure theirs out first.

Reply:
Now that I think about it, I might suggest that engineer move those camera specific macros to the new `camera.h` in #797.